README.md

```diff
@@ -1303,3 +1526,114 @@ To close out a communication session with the head unit, an application sends an
 
 ### 6.2 Closing Specific Services
 If the application doesn't want to completely stop its session, but only wishes to close a specific session it can do so using an `EndService` packet that's service type matches the service that the application is trying to close. The `EndService` packet should include the hash ID in its payload that was contained in the `StartServiceACK` for  that specific service.
+
+## 7. Secured Communication
+
+It is possible to establish a secured and encrypted communication with the system by setting the frame header encryption flag to `1` when starting a new service or by sending another `StartService` with the encryption flag set to `1` when the service is already established (this the required flow for the RPC service). If the authentication is successful, the system will reply with a `StartService ACK` frame with the encryption flag also set to `1` indicating that encrypted data is now accepted. If the authentication fails for some reason, the system will reset the TLS connection and return a `StartService NAK` frame.
+
+Before the encryption of RPC service is enabled (encryption is not available), SDL Core rejects any RPC request with result code `ENCRYPTION_NEEDED` if the RPC needs protection (please see policy updates for which RPCs need protection). SDL Core continues processing an RPC request if the RPC does not need protection. SDL Core sends a notification only if the notification RPC does not need protection.
+
+After the encryption of RPC service is enabled (encryption is available), SDL Core rejects any unencrypted RPC requests with result code `ENCRYPTION_NEEDED` with the unencrypted response if the RPC needs protection. SDL Core continues processing an unencrypted RPC request if the RPC does not need protection and responds with an unencrypted response. SDL Core continues processing an encrypted RPC request if the RPC needs protection and responds with an encrypted response. SDL Core sends an unencrypted notification if the RPC does not need protection. SDL Core sends an encrypted notification if the RPC needs protection. In addition, SDL Core shall continue processing an encrypted RPC request if the RPC does not need protection and responds with an encrypted response.
+
+### 7.1 Authentication
```

Comment:
I'm wondering whether the protocol spec is the best place for all of this info in `7.1`, it might be better to include this in the SDL Overview Guides and link out to it here. It has several references to app and Core configurations that probably aren't really relevant to the protocol

It might still be worth including the TLS handshake diagram here, but most of the other information I feel could be moved to a better place.

Reply:
@jacobkeeler could you advise if [this](https://github.com/smartdevicelink/sdl_core_guides/tree/master/docs/Feature%20Documentation) is an right place for moving out the information from this section?

Reply:
@jacobkeeler updated in https://github.com/smartdevicelink/protocol_spec/pull/41/commits/efddba1d5059d7aa763a2502acb600a726535001

Reply:
@AKalinich-Luxoft I think the best place for this information would be the SDL Overview Guides, since it relates to both the mobile libraries and SDL Core. It might need to be in its own "Protected Services" page for now, as the existing "RPC Encryption" guide doesn't really cover encrypted video and audio services.

Reply:
@jacobkeeler I added a link to SDL Overview Guides in https://github.com/smartdevicelink/protocol_spec/pull/41/commits/b25ef60451a378bf1a7adf7701eaa96481e93454
Also, I opened [PR](https://github.com/smartdevicelink/sdl_overview_guides/pull/80) containing moved changes. The link in protocol_spec will become valid once this PR is merged. Could you please review it as well?